Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/CarbonBlackEndpointStandardV3_description.md

```diff
@@ -0,0 +1,40 @@
+## Carbon Black Cloud Endpoint Standard V3
+ 
+*Carbon Black Cloud Endpoint Standard is the new name for the product formerly called CB Defense.*
+
+### Create Access Levels and API Keys
+
+In this integration we use 2 rest API's with 2 different permissions. One for the policies commands and another for the rest all commands. Therefore, in addition to creating a Live Response API key and Live Response API secret key (for the policies commands), you also need to create a custom API key and a custom API secret key (for the rest all commands), with a custom access level.
```

Comment:
This integration uses two rest APIs with two different permissions, one for the policy commands and another for the rest all commands. Therefore, in addition to creating a Live Response API key and API secret key for the policy commands, you also need to create a custom API key and a custom API secret key for the rest all commands, with a custom access level.